donkeycar/management/makemovie.py

```diff
@@ -1,13 +1,11 @@
 import moviepy.editor as mpy
 from tensorflow.python.keras import activations
 from tensorflow.python.keras import backend as K
-
+import tensorflow as tf
+import cv2
+from matplotlib import cm
 try:
```

Comment:
Thank you so much for reviewing. Yes this can be done without `keras-viz`! And we can definitely remove the try/catch. Some codes use the utilities from `keras.vis` which are `utils.normalize` and `utils.apply_modifications`, but I will implement them directly without importing them from `keras.vis` (they are just simple methods).

Reply:
You're right, I missed the `utils` module. We can have a go at that in a separate change. Your help would be appreciated!